platform/representation/src/gestures/DropGesture.js

```diff
@@ -66,8 +69,21 @@ define(
                     );
                 }
             }
+            
+            function shouldCreateVirtualPanel(domainObject){
+                //
+                return domainObject.useCapability('view').filter(function (view){
+                    return view.key==='plot' && domainObject.getModel().type!== 'telemetry.panel'
+                }).length > 0;
+            }
 
             function dragOver(e) {
+                //Refresh domain object on each dragOver to catch external
+                // updates to the model
+                //Don't use EditableDomainObject for folders, allow immediate persistence
+                editableDomainObject = domainObject.hasCapability('editor') || domainObject.getModel().type==='folder' ? domainObject : new EditableDomainObject(domainObject, $q);
+                actionCapability = editableDomainObject.getCapability('action');
```

Comment:
At the moment always returns a compose action. Perhaps it could return a special 'composePanel' action that encapsulated some of the panel-specific logic.